packages/orderbook/test/js_api/gui.test.ts

```
@@ -877,7 +929,8 @@ ${dotrain}`;
 			assert.equal(deposits[1].amount, '100');
 			assert.equal(deposits[1].address, '0x8f3cf7ad23cd3cadbd9735aff958023239c6a063');
 
-			let guiDeployment: GuiDeploymentCfg = gui.getCurrentDeployment();
+			let result = gui.getCurrentDeployment();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)